src/utils/history-utils.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const getHistory = async (dispatchUserData) => {
+  try {
+    const response = await axios.get('/api/user/history', {
+      headers: { authorization: localStorage.getItem('token') },
```

Comment:
You can consider fetching token from local storage inside your auth context and passing it where needed

Reply:
Makes sense. Will do it once done with the rest.